src/app/components/archive-hero.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classnames from 'classnames-prefix';
+import ArticleHero from './article-hero';
+
+const cx = classnames( 'archive-hero' );
+
+export default function ArchiveHero() {
+    return (
+        <div className={cx('')}>
+            <div className={cx('--primary')}>
```

Comment:
Fain el grid eli 2araftni beh, this is mixing concerns, data and ui. Data shouldn't care how items are laid out.

This will need to be changed, so items are siblings that are laid out by flex or cssgrid.